tools/cloud-build/daily-tests/ansible_playbooks/test-validation/test-gke-kueue-config-a3-mega.yml

```diff
@@ -25,7 +25,7 @@
 - name: Create the topology kueue
   delegate_to: localhost
   ansible.builtin.shell: |
-    array=({{ workspace }}/tools/cloud-build/daily-tests/blueprints/kueue-config-files/tas-queues.yaml)
+    array=({{ workspace }}/tools/cloud-build/daily-tests/blueprints/kueue-config-files/tas-queues-h100.yaml)
```

Comment:
Don't create 3 different files. Figure out a way to pass the value as a parameter to the `tas-queues.yaml` file.